cmd/createcluster.go

```diff
@@ -366,6 +368,15 @@ func validateCreateConfig(ctx context.Context, conf clusterConfig) error {
 		}
 	}
 
+	// check for threshold parameter
+	if conf.Threshold < minThreshold {
+		return errors.New("threshold cannot be smaller than 2", z.Int("threshold", conf.Threshold))
```

Comment:
Don't mention 2 in the message, use `z.Int` param similar to how `create dkg` is made.

Reply:
done